docs/UserGuide.md

```diff
@@ -114,20 +114,20 @@ This detailed guide will walk you through how to make the most of **EduManage**
-|  12   |    Task Index     |                `ti/TASK_INDEX`                 | Should be a positive integer equal to or less than the size of the specified student's task list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| Index |     Parameter     |                     Format                     | Constraints                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|:-----:|:-----------------:|:----------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   1   |      Address      |                  `a/ADDRESS`                   | Can take any values except prefixes already reserved, and should not be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
maybe list out the prefixes it cannot take, such as `a/`, `e/`, `lt/`, `l/`, `n/`, `nt/`, `p/`, `s/`. Do test out to make sure these prefixes cannot work for both add and update